app/models/image_import_job.rb

```diff
@@ -0,0 +1,9 @@
+class ImageImportJob < ManageIQ::Providers::AnsiblePlaybookWorkflow
```

Comment:
This this should live under `ManageIQ::Providers` / `app/models/manageiq/providers`

Reply:
working on it now

Reply:
renamed/moved